install_config/downgrade.adoc

```diff
@@ -114,7 +115,7 @@ remove the following packages:
     atomic-openshift-docker-excluder
 ----
 
-. If you are using external etcd, also remove the *etcd* package:
+. If you use a separate etcd cluster, also remove the *etcd* package:
```

Comment:
@kalexand-rh  
Maybe someone who knows etcd will know this, but are _separate etcd cluster that runs on different nodes,_ and _separate etcd cluster,_ the same?

Reply:
Ah, I see what you mean. I'm going to change one of them. Thanks!